charts/prefect-server/templates/background-services-deployment.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.backgroundServices.runAsSeparateDeployment }}
+{{- include "prefect-server.validatePrefectVersion" . }}
```

Comment:
this calls the validation helper

Reply:
Can we just call the validation helper in `NOTES.txt`? It's been a little while since I've used this, but I think that's where I've called it in the past so it's in a central location instead of being tied to a specific template.

Reply:
Sure!